curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-035.md

```diff
@@ -1,20 +1,38 @@
 ---
-id: 6193fd22d907385efadb59c9
+id: 61fdb04d9939f0a26ca51c2b
 title: Step 35
 challengeType: 0
 dashedName: step-35
 ---
 
 # --description--
 
-The text does not look quite right. Typically, in an accounting sheet, the numeric columns are justified to the right. Give your `span:not(.name)` selector a `text-align` property set to `right`.
+Time to style your table heading. Create an `h1` selector. Give it a `max-width` property set to `37.25rem`, a `margin` property set to `0 auto`, and a `padding` property set to `1.5rem 1.25rem 1.5rem`.
```

Comment:
pretty minor - does the padding need the third property here? `padding: 1.5rem 1.25rem 1.5rem` - I'm fine leaving it.

Reply:
It doesn't - pretty sure the tests will pass without it, even.

Reply:
Updated.